docs/questions/query-builder/expressions/cumulative.md

```diff
@@ -49,47 +49,35 @@ If the sort is changed (while values remain the same), then the cumulative count
 | July     | 5       | 2 + 5 = 7         |
 | November | 4       | 2 + 5 + 4 = 11    |
 
-Metabase will automatically sort the data by the breakout column in ascending order (using the logic that your database uses for the column's data type). If you add a manual sort, it will only apply _after_ cumulative count or sum computes its results. Sorting here has no effect on how Metabase computes the cumulative metric.
-
-If you want to use a different order (for example, sort by the breakout column but in descending order), you can [use SQL](#related-functions).
+When you have only one breakout in the query, Metabase will sort the data by the breakout column in ascending order (using the logic that your database uses for the column's data type). You can add a **Sort** block by the breakout column to change how Metabase accumulates the metric.
 
 ## Cumulative metrics with multiple breakouts
 
-Because cumulative metrics calculate its summaries based on previous rows, Metabase needs to determine what those previous rows are. Cumulative metrics will always order the rows based on the _first_ breakout. So if there are multiple breakout columns specified in **Group by**, Metabase will:
-
-1. Sort by the first breakout column to determine what the previous rows are.
-2. Break out by the first column and any additional columns.
-3. Compute cumulative sums or counts for the additional breakouts over the first column.
-
-This sorting in step 1 only happens in the context of calculating the Cumulative metric.
-
-If you then want to manually sort the results, this sorting only applies _after_ cumulative count or sum computes its results. Sorting here has no effect on how Metabase computes the cumulative metric.
+Because cumulative metrics calculate its summaries based on previous rows, Metabase needs to determine what those previous rows are. Metabase will decide how to compute and display the cumulative metric based on whether you're grouping by a datetime column, and on the order of other breakouts:
```

Comment:
```suggestion
Because cumulative metrics calculate their summaries based on previous rows, Metabase needs to determine what those previous rows are. Metabase will decide how to compute and display the cumulative metric first based on whether you're grouping by a datetime column, then by the order of other breakouts:
```